src/elements/content-sidebar/activity-feed/annotations/useSuppresiveClicks.ts

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+
+export type handlers<T> = {
+    onClick: (event: React.SyntheticEvent<T>) => void;
+    onMouseDown: (event: React.MouseEvent<T>) => void;
+};
+
+export type Props = {
+    isDisabled: boolean;
+    onClick: () => void;
+};
+
+export default function useSuppressiveClicks<T extends HTMLElement>({ isDisabled, onClick }: Props): handlers<T> {
+    const handleClick = (event: React.SyntheticEvent<T>) => {
```

Comment:
Since the logic in this hook is fairly simple and it is only used twice, it may be more readable to duplicate it in each component.

Reply:
Makes sense, I'll look into refactoring the clickable div in `AnnotationActivity` into a component